MyPooDiary/MyPooDiary/Network/Router/FriendsRouter.swift

```diff
@@ -22,20 +23,26 @@ extension FriendsRouter: TargetType {
         switch self {
         case .fetchFriendList:
             return "/friend"
+        case .fetchFriendStory:
+            return "/friend/1"
```

Comment:
지금도 좋지만  명세서에`/:userId`라고 적혀있으면 path 이렇게 해줌 좋을거같아여
이거는 샘플이라 userId가 1밖에 없지만 ..!!

```suggestion
        case .fetchFriendStory(let userId):
            return "/friend\(userId)"
```